config/helpers.php

```diff
@@ -548,22 +548,27 @@ function option(string $key, $default = null)
  * id or the current page when no id is specified
  *
  * @param string|array ...$id
- * @return \Kirby\Cms\Page|null
+ * @return \Kirby\Cms\Page|\Kirby\Cms\Pages|null
+ * @todo reduce to one parameter in 3.7.0 (also change return type)
  */
 function page(...$id)
 {
     if (empty($id) === true) {
         return App::instance()->site()->page();
     }
 
+    if (count($id) > 1) {
+        deprecated('Passing multiple parameters to the `page()` helper has been deprecated. Please use the `pages()` helper instead.');
+    }
+
     return App::instance()->site()->find(...$id);
 }
 
 /**
  * Helper to build page collections
  *
  * @param string|array ...$id
- * @return \Kirby\Cms\Pages
+ * @return \Kirby\Cms\Page|\Kirby\Cms\Pages|null
```

Comment:
I think the return value here should be consistent as well. So when passing a single ID, I'd expect to get a `Pages` collection with a single element.

See https://github.com/getkirby/kirby/pull/3048#pullrequestreview-561332134 for how I'd implement it.

Reply:
That would be another breaking change. So how do we do that to offer a smooth transition?

Reply:
See https://github.com/getkirby/kirby/pull/3048#issuecomment-754237313. As it was never documented, it's technically not a breaking-change.

Of course it's not optimal and a few people may run into this, but I think it's worth it to clean this up. If we documented the `Page` return type explicitly, it would make the situation worse as we would never be able to clean this up.

Reply:
What we could do for the transition: Throw a deprecation warning if the `pages()` helper is called with only one argument and then change the behavior in 3.7.0.

Reply:
I think I'd prefer the latter. This isn't a crucial thing to change for us. So I would make it as smooth as possible.

Reply:
Yes, I agree. We definitely shouldn't break sites if we can avoid it.